### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by dimkouv**
Add CCIP CODEOWNERS.

### CODE REVIEW

**Review by RensR — APPROVED** [review]

**Review by jmank88 — CHANGES_REQUESTED** [review]
I'm not sure that it makes sense to splinter ownership here.

**Comment by dimkouv** [comment]
@jmank88 wouldn't that allow our team to merge our PRs, without requiring your approval?

**Comment by jmank88** [comment]
> @jmank88 wouldn't that allow our team to merge our PRs, without requiring your approval?

Yes but this API needs to stay organized and coordinated as a whole.

File: CODEOWNERS
Comment (patrickhuie19) [inline CODEOWNERS:7]: Could you delete this file and then use CODEOWNERS in the `.github` repo? We want to make that the default location going forward since github looks there first (cc @jmank88)

**Comment by dimkouv** [comment]
> Yes but this API needs to stay organized and coordinated as a whole.
closing then

**Pull request closed without merging**
